### PR TITLE
`element-event` ingestion

### PR DESCRIPTION
**Pull request by CBroz1**
- I started from @kabilar's [main branch](https://github.com/kabilar/workflow-array-ephys) to include docker organization updates. These updates are independent from `element-trial` related changes and could easily be reverted.
- I added ingestion functions for each level of data organization in `element-event` - Recording, Block, Trial, Event, and Alignment
- Fake data imagines an experiment with
   - 4 blocks of 2 types ('light', 'dark')
   - 2 trial types ('ctrl', 'stim')
   - A 'lumens' trial attribute (random int btwn 500 and 1000 for 'light' blocks)
   - 0 to 3 events per trial as imagined gaze or button press ('left', 'right', 'center')
   - Trial duration is fixed, but trial start has some jitter

### CODE REVIEW

File: user_data/events.csv [hunk 1]
```diff
@@ -0,0 +1,53 @@
+subject,session_datetime,trial_id,event_id,event_start_time,event_type
```
Comment (kabilar) [inline user_data/events.csv:1]: Could you please remove `event_id`?  I believe we opted to not use `event_id` and just have `event_start_time`.

File: workflow_array_ephys/ingest.py [hunk 2]
```diff
@@ -115,6 +134,38 @@ def ingest_sessions(session_csv_path='./user_data/sessions.csv', verbose=True):
...
+    csvs = [recording_csv_path, recording_csv_path,
+            block_csv_path, block_csv_path,
+            trial_csv_path, trial_csv_path, trial_csv_path,
+            trial_csv_path,
+            event_csv_path]
+    tables = [event.BehaviorRecording(), event.BehaviorRecording.File(),
+              trial.Block(), trial.Block.Attribute(),
+              trial.TrialType(), trial.Trial(), trial.Trial.Attribute(),
+              trial.BlockTrial(),
+              event.EventType(), event.Event()]
```
Comment (kabilar) [inline workflow_array_ephys/ingest.py:151]: I notice 9 files listed but 10 tables.  Was this intended?  Perhaps I am misunderstanding something.  Thanks.

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @CBroz1.  This is great!  A few minor comments above.

**Comment by CBroz1** [comment]
Thanks for your review @kabilar - I've made the changes to remove event ID and balance out the csv/tables lists in `ingest_events()`

**Review by kabilar — APPROVED** [review]
Thanks @CBroz1.  Great work!

**Comment by CBroz1** [comment]
Thanks! I'll fill in docstrings and other documentation in future commits

**Pull request merged**